test/integration_test/basic_test.go

```diff
@@ -295,6 +293,22 @@ func BasicInstallInCustomNamespace(tc *types.TestCase) func(*testing.T) {
 			}
 		}
 
+		// Create AlertManager secret, if AlertManager is enabled
+		if cluster.Spec.Monitoring != nil {
+			if cluster.Spec.Monitoring.Prometheus != nil {
+				if cluster.Spec.Monitoring.Prometheus.AlertManager != nil {
+					if cluster.Spec.Monitoring.Prometheus.AlertManager.Enabled {
+						alertManagerSecret, err := ci_utils.ParseSpecs("monitoring/alertmanager-secret-custom-namespace.yaml")
```

Comment:
are we able to change the namespace here after parsing instead of creating a new yaml spec?

Reply:
ParseSpecs unfortunately returns `func ParseSpecs(filename string) ([]runtime.Object, error)`, I tried and not sure how to change `namespace` in this `runtime.Object`

Reply:
Done